integration-tests/join/disk/go_test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+cd /root/bin
+echo "clear folders"
+rm -rf /root/bin/.enclave
+rm -rf /root/bin/.cro-storage
+rm -rf /root/bin/.storage
+rm -rf /enclave-storage
+echo "copy binaries"
+mkdir /root/bin
+echo "copy tendermint config"
+cp -Rf /root/disk/bin/* /root/bin
+mkdir /root/.tendermint
+mkdir /root/.tendermint/config
+cp /root/config/*  /root/.tendermint/config
+echo "clear disk"
+/root/bin/tendermint unsafe_reset_all
+sleep 2 
+source /root/disk/prepare.sh
+source /opt/sgxsdk/environment
+source /root/.cargo/env
+echo "sgx mode=" $SGX_MODE
+echo "network id=" $NETWORK_ID
+echo "spid = " $SPID
+echo "ias_api_key = " $IAS_API_KEY 
```

Comment:
again, as secrets, they shouldn't be printed out

Reply:
removed